src/metabase/api/dashboard.clj

```diff
@@ -764,7 +768,7 @@
                         :status-code     400})))
      (case (:source_type param)
        "static-list" (static-parameter-values param query)
-       "card"        (parameter-card/values-for-dashboard dashboard param-key query)
+       "card"        (card-parameter-values param query)
        (chain-filter dashboard param-key constraint-param-key->value query)))))
```

Comment:
Also something from before, but this should explicitly handle `nil`

Reply:
thanks. handled in https://github.com/metabase/metabase/pull/27343/commits/373301925d5902d51d0ab1f5e9f2b8f55e496f08